Sources/KituraSampleRouter/RouterCreator.swift

```diff
@@ -20,7 +20,8 @@ import Foundation
 
 import Kitura
 import KituraMustache
-import KituraStencil
+import KituraStencil // required for using StencilTemplateEngine
```

Comment:
@ianpartridge this is the whitespace on my keyboard. Could you please the following check for me - checkout the branch https://github.com/IBM-Swift/Kitura-Sample/tree/issue_907, remove these whitespaces and add them, perform `git diff` and see if the whitespaces on your keyboard are different?